migrations/versions/patch_DEEP_FUTURE_last_unfollowedup.py

```diff
@@ -0,0 +1,37 @@
+# Migration script generated for patch_DEEP_FUTURE_last_unfollowedup
+revision = '7542b481-21cb-483c-b72a-6c1502375a65'
+down_revision = '17f3b60d-0777-4d0e-bb32-82e670b573a7'
```

Comment:
Just curious, where do you get these values from?

Reply:
These are UUIDs, built into the FHIR Migrations framework.
The most recent migration run is stored in a FHIR Basic object.  To generate these automatically, run the following:
`dc exec <service> migrate <name-of-migration-script-to-generate>`

For example, I ran:
` dc exec messagingservice flask migrate patch_DEEP_FUTURE_last_unfollowedup`

and as it found the down_revision in the DB, fabricated a shell script in `migrations/version/<name>.py` which included those and boiler plate functions to extend.